docs/ssr.mdx

```diff
@@ -41,6 +41,46 @@ import { renderToString } from 'react-dom/server'
 import createEmotionServer from '@emotion/server/create-instance'
 import createCache from '@emotion/cache'
 
+const key = 'custom'
+const cache = createCache({ key })
+const { extractCriticalToChunks, constructStyleTags } = createEmotionServer(cache)
+
+let element = (
+  <CacheProvider value={cache}>
+    <App />
+  </CacheProvider>
+)
+
+let { html, styles } = extractCriticalToChunks(renderToString(element))
+
+res
+  .status(200)
+  .header('Content-Type', 'text/html')
+  .send(`<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My site</title>
+    ${constructStyleTags({ html, styles })}
+</head>
+<body>
+    <div id="root">${html}</div>
+
+    <script src="./bundle.js"></script>
+</body>
+</html>`);
+```
+
+#### Deprecated
```

Comment:
For now, this won't get deprecated - the recommendation would be that:
- `extractCriticalToChunks` should be used when using `@emotion/react`
- `extractCritical` should be used when using `@emotion/css`

Reply:
Updated, you can re-check the wording.